buildspec.yml

```diff
@@ -12,6 +12,7 @@ phases:
       - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION)
       - pip install -r src/requirements.txt
       - bash src/setup.sh $FRAMEWORK
+      - export ROOT_FOLDER_PATH=$(pwd)
```

Comment:
I don't think `$HOME` would necessary be the right env variable here, since this file is executed on CB. `CODEBUILD_SRC_DIR` should already be set, and you should be able to leverage that for most of your needs

Reply:
Removed. Followed Arjuna's suggestions.